src/queries/find/stronglyConsistent.ts

```diff
@@ -0,0 +1,40 @@
+import { IDynamoDbManager } from "../../interfaces/iDynamodbManager";
+import { Execute } from "./execute";
+import { Using } from "./using";
+import { Order } from "./order";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import { Resume } from "./resume";
+
+export class StronglyConsistent
```

Comment:
The current PR only add stronglyConsistent support to find().  Could you add this to get() and list() as well?

Reply:
I think you meant this only works for `list()`, and you want me to support `find()` and `get()` as well?

I think both Find and List are supported _as is_ -- `find()` strictly requires `where()` before you can `execute()`, and `where()` supports `stronglyConsistent()`.

As for `get()`, the manager simply wont support it as is... We'd need to re-write getOne to not use the GSI. I can push up what I have, but it will get stuck at the `getOne()` function in `dynamoDbManager.ts`

Reply:
Any update on your previous comment here?

Reply:
I have working acceptance tests for `getOne()` (since this doesn't involve mocks, all the existing tests worked, and I only needed to add a few). I'm looking at unit tests now, but before I fix them, I'm debating if `getById()` needs to exist anymore? I see it used internally, but it looks so similar to `getOne()` that I'm thinking we don't need `getById()` anymore? Am I missing something?

Reply:
We need that one so the code can get all representations of the id back.

Reply:
Okay, I left GetById() unchanged, so should be fine then